src/node_url.cc

```diff
@@ -1418,6 +1419,66 @@ namespace url {
                             v8::NewStringType::kNormal).ToLocalChecked());
   }
 
+  MaybeLocal<Value> URL::ToObject(Environment* env) {
+    Isolate* isolate = env->isolate();
+    Local<Context> context = env->context();
+    HandleScope handle_scope(isolate);
+    Context::Scope context_scope(context);
+
+    const Local<Value> undef = Undefined(isolate);
+    if (context_.flags & URL_FLAGS_FAILED)
+      return undef;
```

Comment:
Either return a `Local<Value>` as @addaleax suggested, or make this return `MaybeLocal<Value>()`.